packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -190,9 +198,23 @@ function NavigationMenuItemEdit( {
 					className="wp-block-navigation-menu-item__content"
 					value={ label }
 					onChange={ ( labelValue ) => setAttributes( { label: labelValue } ) }
-					placeholder={ __( 'Add item…' ) }
+					placeholder={ itemLabelPlaceholder }
 					withoutInteractiveFormatting
 				/>
+
+				{ isLinkOpen &&
+					<LinkControl
```

Comment:
This doesn't seem to be rendering in the correct location when there's a list of sub menu-items:
![Screen Shot 2019-11-05 at 2 27 17 pm](https://user-images.githubusercontent.com/677833/68183970-ca167900-ffd8-11e9-8d6f-0c2a472429db.png)

Popovers render relative to their parent, so will probably need a wrapping `div` element around the `RichText` and move the `LinkControl` into that `div`.

Reply:
Great spot. I've added that wrapper and it seems to be working:

![Screen Capture on 2019-11-05 at 09-40-26](https://user-images.githubusercontent.com/444434/68196346-611b0b00-ffb0-11e9-9f2d-6d55c7e9389a.gif)